build/yamls/base/conf/antrea-agent.conf

```diff
@@ -120,3 +123,13 @@ featureGates:
 # Provide the address of Kubernetes apiserver, to override any value provided in kubeconfig or InClusterConfig.
 # Defaults to "". It must be a host string, a host:port pair, or a URL to the base of the apiserver.
 #kubeAPIServerOverride: ""
+
+# The virtual IP for NodePort Service support. It must be a link-local IP otherwise the Agents will report error.
+#nodePortVirtualIP: 169.254.169.110
+
+# The virtual IPv6 for NodePort Service support. It must not be a link-local IP otherwise the Agents will report error.
```

Comment:
Why it cannot be link-local?

Reply:
The IPv6 forwarding is stricter than IPv4, and link-local IP is not forwardable.

Reply:
Ok.